app/src/main/java/fr/free/nrw/commons/contributions/ContributionBoundaryCallback.kt

```diff
@@ -16,11 +16,11 @@ import javax.inject.Named
  */
 class ContributionBoundaryCallback @Inject constructor(
     private val repository: ContributionsRepository,
-    private val sessionManager: SessionManager,
     private val mediaClient: MediaClient,
     @param:Named(CommonsApplicationModule.IO_THREAD) private val ioThreadScheduler: Scheduler
 ) : BoundaryCallback<Contribution>() {
     private val compositeDisposable: CompositeDisposable = CompositeDisposable()
+    var userName: String?=null
```

Comment:
Hi @ashishkumar468 , I have some style reviews. there should be spaces before and after equal operands. It is missing in some places, like here